test/konfig.bats

```diff
@@ -121,6 +121,37 @@ load common
   [[ $(check_kubeconfig 'testdata/config123') = 'same' ]]
 }
 
+@test "import single config from stdin and print to stdout" {
+  use_config config1
+  run bash -c "KUBECONFIG=testdata/config-2 kubectl config view --flatten | ${COMMAND} import -i"
+  echo "$output"
+  [[ "$status" -eq 0 ]]
+  [[ $(check_fixture 'testdata/config12-flat' "$output") = 'same' ]]
+}
+
+@test "import single config from stdin" {
+  use_config config1
+  run bash -c "KUBECONFIG=testdata/config-2 kubectl config view --flatten | ${COMMAND} import -i --save"
```

Comment:
```suggestion
  run bash -c "cat testdata/config-2 | ${COMMAND} import -i --save"
```

Is that possible? I'd like to avoid setting `KUBECONFIG`, because it might interfere with `kconfig`.

[same above]

Reply:
As config-2 refers to credenticals directory, test didn't work. So I used kubectl. But I agree with you.
Therefore, I use config3 instead now.